### PR TITLE
Add description for /fuckoff

### PR DESCRIPTION
**Pull request by robingall2910**
For issue #813

and a fix from #833

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_fuckoff.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SENIOR, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "You'll never even see it coming.", usage = "/<command>")
+@CommandParameters(description = "You'll never even see it coming.", usage = "/fuckoff [on <range> | off]")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_fuckoff.java]: When a group has OR behavior, it should be detonated with `<` and `>`.

**Comment by OxLemonxO** [comment]
you don't need to put /fuckoff, you can keep /<command>

**Comment by JeromSar** [comment]
@OxLemonxO is correct.

**Comment by xDestroyer217** [comment]
Hope I'm correct. @JeromSar

**Comment by JeromSar** [comment]
@xDestroyer217 Yes, perfect. Thank you!

**Comment by OxLemonxO** [comment]
/<command> <on [range] | off> not
<on [range] | off>

**Comment by JeromSar** [comment]
@xDestroyer217 Sorry, just saw that @OxLemonxO is correct here. Please fix.

**Comment by xDestroyer217** [comment]
Dammit, right.

**Comment by xDestroyer217** [comment]
Updated. @JeromSar

**Comment by Wild1145** [comment]
Thank you for your contribution. Sadly we have decided to close this issue as there has been no recent activity to this pull request. If you wish to continue working on this PR please re-open the issue and ensure that all requested changes are made.

**Pull request closed without merging**